base/doc.dtx

```diff
@@ -57,7 +57,7 @@
 %
 %
 %% Package `doc' to use with LaTeX 2e
-%% Copyright (C) 1989-2022 Frank Mittelbach, all rights reserved.
+%% Copyright (C) 1989-2023 Frank Mittelbach, all rights reserved.
```

Comment:
This copyright may need an update to be covered by custom `update_tag()` in this repo.

Reply:
It's in the top-level `build-config.lua` file.

Reply:
Oh, in
```lua
update_tag = update_tag_base
-- ...
dofile (maindir .. "/build-config.lua")
```
`update_tag` is re-assigned twice, hence the second one done in `build-config.lua` wins.
https://github.com/latex3/latex2e/blob/cc704066a05e5fb1563786558db0cd5f3e91e0ec/build-config.lua#L96

Thus the line `update_tag = update_tag_base` in `base/build.lua` can be removed.